braid/relax.c

```diff
@@ -36,18 +36,88 @@ _braid_FCRelax(braid_Core  core,
    braid_Real     *CWts     = _braid_CoreElt(core, CWts);
    _braid_Grid   **grids    = _braid_CoreElt(core, grids);
    braid_Int       ncpoints = _braid_GridElt(grids[level], ncpoints);
-
+   
    braid_BaseVector  u, u_old;
    braid_Real        CWt;
    braid_Int         flo, fhi, fi, ci;
    braid_Int         nu, nrelax, interval;
 
+   /* Required for Richardson */
```

Comment:
@jbschroder I would like to think about how we might reduce the impact of Richardson on readability of this routine (and maybe some others), but that can wait until a later date, after we merge this pull request.

Reply:
@rfalgout Yes, I had some similar thoughts.  But, I frankly started to run out of steam on this PR.  We could try writing some helper functions for relax.c and restrict.c, perhaps.